packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -19,13 +19,16 @@ import { pipe, useCopyToClipboard } from '@wordpress/compose';
  * Internal dependencies
  */
 import BlockActions from '../block-actions';
+import { privateApis as blockEditorPrivateApis } from '../../private-apis';
 import BlockHTMLConvertButton from './block-html-convert-button';
 import __unstableBlockSettingsMenuFirstItem from './block-settings-menu-first-item';
 import BlockSettingsMenuControls from '../block-settings-menu-controls';
 import BlockParentSelectorMenuItem from './block-parent-selector-menu-item';
 import { store as blockEditorStore } from '../../store';
 import { unlock } from '../../lock-unlock';
 
+const { __unstableCommentIconFill } = unlock( blockEditorPrivateApis );
```

Comment:
This will probably cause errors: when using a private API internally in the same package, you should just import them directly.